src/dmd/dsymbol.d

```diff
@@ -447,16 +449,7 @@ extern (C++) class Dsymbol : ASTNode
     }
 
     /// ditto
-    final inout(Dsymbol) pastMixinAndNspace() inout
-    {
-        //printf("Dsymbol::pastMixin() %s\n", toChars());
-        auto nspace = isNspace();
-        if (!(nspace && nspace.mangleOnly) && !isTemplateMixin() && !isForwardingAttribDeclaration())
-            return this;
-        if (!parent)
-            return null;
-        return parent.pastMixinAndNspace();
-    }
+    alias pastMixinAndNspace = pastMixin;
```

Comment:
is pastMixinAndNspace even used anymore, it can probably just be removed.

Reply:
Actually most of the change related to `toParent3` can be reverted.
That's quite a lot of changes though, I was considering putting them in an extra commit, what do you think ?